firewall.rst

```diff
@@ -43,6 +43,47 @@ In this configuration, the firewall regulates traffic between the WAN and LAN zo
 
 Default zones can't be deleted but the network administrator can change existing policies or create new zones. 
 
+Guests and DMZ zones
+--------------------
+
+In addition to the default zones, the firewall can be configured with additional zones to accommodate specific network requirements.
+Two common examples are the Guests and DMZ (Demilitarized Zone) zones.
+Sometimes Guests zone is also named as vlue zone while DMZ zone is also named as orange zone.
+
+Guests zone (blue)
+^^^^^^^^^^^^^^^^^^^
+
+Guests zone represents a network segment for guest devices, such as visitors or temporary users.
+This zone is typically isolated from the LAN zone to prevent unauthorized access to internal resources.
+But it is allowed to access the WAN zone.
+
+To create a Guests zone, follow these steps:
+
+- access the ``Zones & policies`` section
+- click on the ``Add zone`` button
+- enter **guests** inside the ``Name`` field: plesase note that the name is case sensitive and must be in lowercase, in this case the zone will be highlighted in blue
```

Comment:
There's been some talks about naming the zone "guest" instead of guests.

Reply:
Where?